projects/ngx-charts-on-fhir/src/lib/fhir-chart/range-selector/range-selector.component.ts

```diff
@@ -14,11 +14,16 @@ export class RangeSelectorComponent {
   layers?: DataLayer[];
   maxDate: Date | string;
   minDate: Date | string;
-  isMatGroupFocus: boolean = true;
-
+  _isMatGroupFocus: boolean = true;
+  @Input() 
+  set isMatGroupFocus(isMatGroupFocus: boolean){
+    this._isMatGroupFocus = isMatGroupFocus;
+  }
```

Comment:
why do we need a setter for this instead of a simple property? the setter is not doing anything except setting the value.